apps/wallet-mobile/src/features/Swap/useCases/StartSwapScreen/ListOrders/helpers.ts

```diff
@@ -0,0 +1,601 @@
+  'multiplyInteger-memory-arguments-intercept': 0,
+  'multiplyInteger-memory-arguments-slope': 1,
+  'nullList-cpu-arguments': 60091,
+  'nullList-memory-arguments': 32,
+  'quotientInteger-cpu-arguments-constant': 196500,
+  'quotientInteger-cpu-arguments-model-arguments-intercept': 453240,
+  'quotientInteger-cpu-arguments-model-arguments-slope': 220,
+  'quotientInteger-memory-arguments-intercept': 0,
+  'quotientInteger-memory-arguments-minimum': 1,
+  'quotientInteger-memory-arguments-slope': 1,
+  'remainderInteger-cpu-arguments-constant': 196500,
+  'remainderInteger-cpu-arguments-model-arguments-intercept': 453240,
+  'remainderInteger-cpu-arguments-model-arguments-slope': 220,
+  'remainderInteger-memory-arguments-intercept': 0,
+  'remainderInteger-memory-arguments-minimum': 1,
+  'remainderInteger-memory-arguments-slope': 1,
+  'serialiseData-cpu-arguments-intercept': 1159724,
+  'serialiseData-cpu-arguments-slope': 392670,
+  'serialiseData-memory-arguments-intercept': 0,
+  'serialiseData-memory-arguments-slope': 2,
+  'sha2_256-cpu-arguments-intercept': 806990,
+  'sha2_256-cpu-arguments-slope': 30482,
+  'sha2_256-memory-arguments': 4,
+  'sha3_256-cpu-arguments-intercept': 1927926,
+  'sha3_256-cpu-arguments-slope': 82523,
+  'sha3_256-memory-arguments': 4,
+  'sliceByteString-cpu-arguments-intercept': 265318,
+  'sliceByteString-cpu-arguments-slope': 0,
+  'sliceByteString-memory-arguments-intercept': 4,
+  'sliceByteString-memory-arguments-slope': 0,
+  'sndPair-cpu-arguments': 85931,
+  'sndPair-memory-arguments': 32,
+  'subtractInteger-cpu-arguments-intercept': 205665,
+  'subtractInteger-cpu-arguments-slope': 812,
+  'subtractInteger-memory-arguments-intercept': 1,
+  'subtractInteger-memory-arguments-slope': 1,
+  'tailList-cpu-arguments': 41182,
+  'tailList-memory-arguments': 32,
+  'trace-cpu-arguments': 212342,
+  'trace-memory-arguments': 32,
+  'unBData-cpu-arguments': 31220,
+  'unBData-memory-arguments': 32,
+  'unConstrData-cpu-arguments': 32696,
+  'unConstrData-memory-arguments': 32,
+  'unIData-cpu-arguments': 43357,
+  'unIData-memory-arguments': 32,
+  'unListData-cpu-arguments': 32247,
+  'unListData-memory-arguments': 32,
+  'unMapData-cpu-arguments': 38314,
+  'unMapData-memory-arguments': 32,
+  'verifyEcdsaSecp256k1Signature-cpu-arguments': 35892428,
+  'verifyEcdsaSecp256k1Signature-memory-arguments': 10,
+  'verifyEd25519Signature-cpu-arguments-intercept': 57996947,
+  'verifyEd25519Signature-cpu-arguments-slope': 18975,
+  'verifyEd25519Signature-memory-arguments': 10,
+  'verifySchnorrSecp256k1Signature-cpu-arguments-intercept': 38887044,
+  'verifySchnorrSecp256k1Signature-cpu-arguments-slope': 32947,
+  'verifySchnorrSecp256k1Signature-memory-arguments': 10,
+})
+
+export const assertRequired = <T>(value: T | undefined, message: string): T => {
+  if (value === undefined) throw new Error(message)
+  return value
+}
+
+export const fixScriptHash = async (tx: CSL_TYPES.Transaction) => {
```

Comment:
You can recalculate script hash without building tx by tx builder. There is function `hash_script_data` that you can use